src/odemis/driver/andorcam2.py

```diff
@@ -2083,7 +2085,9 @@ def _update_settings(self):
         # "some settings", so check one last time that it's really possible to use
         # software trigger. Or alternatively, maybe the previous settings didn't
         # allow software trigger, but the new ones do.
-        if synchronized and (trigger_mode == TRIG_SW) != self.IsTriggerModeAvailable(AndorV2DLL.TM_SOFTWARE):
+        if (self._supports_soft_trigger and synchronized and
+            (trigger_mode == TRIG_SW) != self.IsTriggerModeAvailable(AndorV2DLL.TM_SOFTWARE)
+           ):
```

Comment:
Would it be interesting to log which conditions are not met to enable software trigger so that the user might activate the required settings to achieve this?